web-marketplace/src/pages/ProjectMetadata/ProjectMetadata.SelectedForm.tsx

```diff
@@ -25,16 +22,17 @@ export const ProjectMetadataSelectedForm = ({
   onPrev,
   creditClassId,
 }: Props): JSX.Element => (
+  // const isVCS = !!creditClassId && isVCSCreditClass(creditClassId);
```

Comment:
Since this component wasn't doing anything I used `MetadataForm` in the parent directly and left this comment to use in the parent in case we need it again.